### PR TITLE
Add pipeline for pushing a Docker image

### PR DESCRIPTION
**Pull request by giorgiosironi**
The pull request only builds the image, https://alfred.elifesciences.org/job/image/job/image-cermine/ should push it once merged.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        commit = elifeGitRevision()
+    }
+
+    elifeOnNode(
+        {
+            stage 'Build images', {
+                checkout scm
+                sh './build.sh'
+            }
+
```
Comment (giorgiosironi) [inline Jenkinsfile:14]: any smoke test that can be added here? `docker run elifesciences/cermine /bin/something`?

Comment (de-code) [reply Jenkinsfile:14]: We could run the container (as a server) and then request a URL or post to the API. (Might be worth to add a healthcheck although it does not need to be production quality at this point)

Comment (giorgiosironi) [reply Jenkinsfile:14]: If you have an example request to use and know which port the server listens on, I can add it

**Review by de-code — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (de-code) [reply Jenkinsfile:14]: `curl -X POST --data "reference=dummy reference" http://localhost:8072/parse.do` (the server is internally listening on port 8080, which the _run.sh_ script maps to 8072 - might put that port into _.env_)

**Comment by giorgiosironi** [comment]
Takes ~30-40 seconds for the container to start, so it was easier to add an healthcheck than to reimplement a polling loop.

**Comment by giorgiosironi** [comment]
https://alfred.elifesciences.org/job/image/job/image-cermine/1/console is also green.

**Comment by de-code** [comment]
Great, thank you.

**Pull request merged**